schema/install_schema.go

```diff
@@ -11,7 +11,7 @@ type InstallSchema struct {
 	BindMounts          []string       `json:"bind_mounts,omitempty"`
 	Bundles             []BundleSchema `json:"bundles,omitempty" description:"Add bundles in runtime"`
 	NoFormat            bool           `json:"no_format,omitempty"`
-	Device              string         `json:"device,omitempty" pattern:"^(auto|/|(/[a-zA-Z0-9_-]+)+)$" description:"Device for automated installs" examples:"[\"auto\",\"/dev/sda\"]"`
+	Device              string         `json:"device,omitempty" pattern:"^(auto|/dev/.+)$" description:"Device for automated installs" examples:"[\"auto\",\"/dev/sda\"]"`
```

Comment:
It doesn't seem to work with escaped slashes `\/` but it works like this so... :shrug: